csuntechlab/iSTART
# STR-84 -- FE-Admin Navigation Bar

# Description
'Write a meaningful description of your task'
  Make the Admin Navigation Bar only include logout for the time being
# Testing
'What does the reviewer need to do to test your PR?'
  1. Login as `steve@csun.edu` then click on the navbar to see that only `logout` is accessible
  2. Login as `april.feldman` then click on the navbar to see all the fields appear
# Installation
'Are there any dependencies the reviewer needs to download?'
There are no dependencies you need to download
